.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
@EugeneKuptsov You should avoid commit IDE files. Could you add VS code files in .gitignore, and remove it from repository?